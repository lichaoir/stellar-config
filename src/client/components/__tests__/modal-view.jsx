const React = require('react');
const ModalView = require('../modal-view.jsx');
const renderer = require('react-test-renderer');

test('Modal view can be rendered', () => {
  const component = renderer.create(
    <ModalView>content</ModalView>
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
