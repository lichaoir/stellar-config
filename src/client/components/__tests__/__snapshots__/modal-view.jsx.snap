// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component modal-view wraps its children 1`] = `
<div
  className="modal is-active"
>
  <div
    className="modal-background"
  />
  <div
    className="modal-content"
  >
    content
  </div>
</div>
`;
